utilities/requirements.txt

```
@@ -1,2 +1,6 @@
 PySimpleGUI==4.60.3
 Pillow==9.3.0
+psutil==5.9.3
```

```suggestion
psutil==5.9.3
pyqt5==5.15.9
opencv-python==4.7.*
numpy==1.24.1
```

Make these 1:1 with examples (opencv-python should be contrib, pyqt5 should be the same, etc...)

---

Changed: https://github.com/luxonis/depthai-python/pull/782/commits/e914e27603118d383b524efd31475e813cb12609